app/services/atmosphere/cloud/vm_creator.rb

```diff
@@ -39,6 +52,30 @@ def execute
 
     attr_reader :user_data
 
+    def set_azure_endpoints(params)
```

Comment:
Assignment Branch Condition size for set_azure_endpoints is too high. [20.71/15]